Damola/LESS/home.less

```diff
@@ -3,7 +3,7 @@
     width: 100%;
     margin: 0 auto;
     @media @mobile {
-        max-width: 800px;
+        max-width: 100%;
         
```

Comment:
There's an extra bit of white-space here; if you're interested, you can use [prettier](https://prettier.io) to format your code quickly and easily. It'll also show you a consistent way to format your CSS, JavaScript (in the future), and more. Code formatting tools can be great and are commonly used when working with other developers.